MAINTAINING.md

```diff
@@ -41,31 +36,39 @@ We try to create and foster a community around Amundsen. We do this by:
 
 - Answering questions from members of the community
 - Triaging Github issues, adding the proper [labels][labels] to new tickets
-- Closing stale issues and feature requests
 - Keeping the community informed by ensuring that we add communications regularly with the new features
-- Ensuring that the documentation, as well as the documentation site, is kept up to
-  date
+- Ensuring that the documentation, as well as the documentation site, is kept up to date
 - Doing code reviews for other maintainers and the community
 - Reviewing [RFCs][rfcs] and shaping the future of the project
 
 [labels]: https://github.com/amundsen-io/amundsen/labels
 [rfcs]: https://github.com/amundsen-io/rfcs
 
-## Workflow
 
-We generally follow [GitHub Flow]. The `master` branch is the main line, and all
+## Issue management
+
+The triage and issue managing process will look like this:
+
+1. New issues will automatically added the "Status: Needs Triage".
+2. Once maintainers review the issue, they will remove the "Status: Needs Triage" label, add new ones and close the issue.
+3. The Amundsen community will upvote the issues they want to be addressed by viewing the [features](https://github.com/amundsen-io/amundsen/issues?q=is%3Aissue+is%3Aclosed+sort%3Areactions-%2B1-desc+label%3Atype%3Afeature+) and [bug](https://github.com/amundsen-io/amundsen/issues?q=is%3Aissue+is%3Aclosed+sort%3Areactions-%2B1-desc+label%3Atype%3Abug+) lists and reacting with a 👍.
```

Comment:
2 notes:
- do we have a min number of upvote/downvotes to move or discard a feature request? 
- As maintainers, we can also vote on any issue, right?